pkgs/development/r-modules/default.nix

```diff
@@ -1803,10 +1803,23 @@ let
       '';
     });
 
-    torch = old.torch.overrideAttrs (attrs: {
-      preConfigure = ''
-        patchShebangs configure
-      '';
```

Comment:
Am I to understand that the original package was source-built? If so, I think we should keep a source-built `torch` and add a separate `torch-bin` attribute for the binary package?

Reply:
> was indeed source built but didn't provide the torch library, it suggested users to download and install it on first run

oh